src/libs/OptionsListUtils.js

```diff
@@ -558,7 +558,7 @@ function createOption(accountIDs, personalDetails, report, reportActions = {}, {
         result.phoneNumber = personalDetail.phoneNumber;
     }
 
-    result.text = reportName;
+    result.text = reportName || LocalePhoneNumber.formatPhoneNumber(personalDetail.login);
```

Comment:
Yes, that works.

We can use [here](https://github.com/Expensify/App/blob/358c3bf4fd161c51cf4ea00110576af3afe03ce9/src/libs/OptionsListUtils.js#L532) this
```js
reportName = ReportUtils.getDisplayNameForParticipant(accountIDs[0]) || LocalePhoneNumber.formatPhoneNumber(personalDetail.login);
```
We cannot wrap the `ReportUtils.getDisplayNameForParticipant(accountIDs[0])` with `LocalePhoneNumber(formatPhoneNumber` directly because the personal details for this accountID are not merged to Onyx and [this here](https://github.com/Expensify/App/blob/5e01376b6648410c95bcaf0ee487d6e8cde259b2/src/libs/ReportUtils.js#L1262-L1268) returns `''`

Reply:
Done.